tls/s2n_cipher_suites.h

```diff
@@ -165,4 +166,5 @@ int s2n_set_cipher_as_client(struct s2n_connection *conn, uint8_t wire[S2N_TLS_C
 int s2n_set_cipher_as_sslv2_server(struct s2n_connection *conn, uint8_t *wire, uint16_t count);
 int s2n_set_cipher_as_tls_server(struct s2n_connection *conn, uint8_t *wire, uint16_t count);
 bool s2n_cipher_suite_requires_ecc_extension(struct s2n_cipher_suite *cipher);
+struct s2n_kem_preferences;
 bool s2n_cipher_suite_requires_pq_extension(struct s2n_cipher_suite *cipher);
```

Comment:
Why the need to forward declare s2n_kem_preferences here? I don't see it used

Reply:
ah, i think this was needed in the prior revision where I'd used that type in `s2n_cipher_suite_requires_pq_extension`'s signature. now that we've removed that parameter we can remove the forward declaration.